### PR TITLE
Update simple load test working

### PR DESCRIPTION
**Pull request by minsulee2**
- Add simple json rpc requester
- Update simple-load-test which has not worked for a long time.

### CODE REVIEW

File: tools/json-rpc-test/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash');
```
Comment (platfowner) [inline tools/json-rpc-test/index.js:1]: json-rpc-test sounds like somewhat broad (note that most tools are actually using json rpc).
what about more specific directory names like 'block'?

File: tools/json-rpc-test/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  ).then((resp) => {
+    return _.get(resp, 'data.result.result');
+  }).catch((err) => {
+    console.error(`Failed to send get request: ${err}`);
+    return null;
+  });
+}
+
+const main = async () => {
```
Comment (platfowner) [inline tools/json-rpc-test/index.js:22]: could you support usage? (ref: tools/transfer/sendTransferTx.js)

Comment (platfowner) [reply tools/json-rpc-test/index.js:22]: also could you give more specific name to this file? (e.g. ainGetLastBlock.js)

File: tools/simple-load-test/index.js [hunk 3]
```diff
@@ -3,18 +3,16 @@
...
+
 const delay = (time) => new Promise((resolve) => setTimeout(resolve, time));
-const testPath = '/apps/loadtest';
-const ainPrivateKey = '4207f5dcacb1b601d3a1f8cb10afaca158f6ebe383c0b30d02b39f8d2060cce3';
-const ainAddress = '0xF2be7f1356347a8960630c112AcB6Da61eE94632';
-const TIMEOUT_MS = 10 * 1000;
+const appName = 'loadtest';
+const testPath = `/apps/${appName}`;
+const ainPrivateKey = 'b22c95ffc4a5c096f7d7d0487ba963ce6ac945bdc91c79b64ce209de289bec96';
+const ainAddress = '0x00ADEc28B6a845a085e03591bE7550dd68673C1C';
```
Comment (platfowner) [inline tools/simple-load-test/index.js:14]: what about moving these params to 'config.js'?

File: tools/json-rpc-test/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash');
+const axios = require('axios');
+const { JSON_RPC_METHODS } = require('../../json_rpc/constants');
+
+const queryOnNode = (method, params) => {
+  return axios.post(
+    'http://localhost:8081/json-rpc',
```
[the remark this answers is not in the document]
Comment (minsulee2) [reply tools/json-rpc-test/index.js:7]: Done!

File: tools/json-rpc-test/index.js (hunk 2 above)
Comment (minsulee2) [reply tools/json-rpc-test/index.js]: Updated.

File: tools/json-rpc-test/index.js (hunk 1 above)
Comment (minsulee2) [reply tools/json-rpc-test/index.js]: This was written when I tested json-rpc with express-rate-limit and I simply used it to test the rate limit is working. What do you think keeping it as 'json-rpc' ?

**Comment by minsulee2** [comment]
PTAL. All addressed!

File: tools/json-rpc-test/index.js (hunk 1 above)
Comment (platfowner) [reply tools/json-rpc-test/index.js]: i see. then what about 'json-rpc-get' as most other tools are also using json-rpc set APIs like sendSignedTransaction()?

**Comment by platfowner** [comment]
could you update the title of this PR?

**Comment by minsulee2** [comment]
@platfowner Done!!

File: tools/json-rpc-test/index.js (hunk 1 above)
Comment (platfowner) [reply tools/json-rpc-test/index.js]: @minsulee2 ping~

Comment (minsulee2) [reply tools/json-rpc-test/index.js]: Oh, I missed it. Updated!

**Comment by minsulee2** [comment]
@platfowner PTAL. updated missed one as well :)

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for the fix and the nice tools!

**Comment by minsulee2** [comment]
Thanks! Merging now!

**Pull request merged**